actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -309,6 +290,19 @@ def content_tag(name, content_or_options_with_block = nil, options = nil, escape
         end
       end
 
+      # Returns a string of class names built from +args+.
+      #
+      # ==== Examples
+      #   class_names("foo", "bar")
+      #    # => "foo bar"
+      #   class_names({ foo: true, bar: false })
+      #    # => "foo"
+      #   class_names(nil, false, 123, "", "foo", { bar: true })
+      #    # => "foo bar"
```

Comment:
@joelhawksley The output mentioned in the docs in wrong. I tried it in one of the application and integer are added to the output string. 

`123.present?` will not return `false` here(https://github.com/rails/rails/pull/37918/files#diff-3d863df5229eeafadb6df53160605a5bR348)

```
123 foo bar
```